lib/usd/hdMaya/adapters/proxyAdapter.cpp

```diff
@@ -18,18 +18,18 @@
 #include <maya/MTime.h>
 #include <maya/MGlobal.h>
 
-#if WANT_UFE_BUILD
-#include <ufe/rtid.h>
-#include <ufe/runTimeMgr.h>
-#endif // WANT_UFE_BUILD
-
-#include <mayaUsd/nodes/proxyShapeBase.h>
-
 #include <hdMaya/adapters/adapterRegistry.h>
 #include <hdMaya/debugCodes.h>
 #include <hdMaya/delegates/proxyDelegate.h>
 #include <hdMaya/delegates/sceneDelegate.h>
 
+#include <mayaUsd/nodes/proxyShapeBase.h>
```

Comment:
Combine into one group.

Reply:
Thanks @mattyjams . Consider it done 6b66abaa8